components/modal/modal.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>modal</title>
+    <link rel="stylesheet" href="modal.css">
+</head>
+<script src="https://kit.fontawesome.com/6b52d0ce66.js" crossorigin="anonymous"></script>
```

Comment:
I personally like to keep the script tag before closing-head tag or before closing-body tag